network.tf

```diff
@@ -29,20 +36,51 @@ resource "google_compute_router_nat" "advanced-nat" {
   name                               = "${var.project_id}-nat${var.cluster_name_suffix}"
   project                            = var.project_id
   router                             = google_compute_router.router.name
-  region                             = local.region
-  nat_ip_allocate_option             = "AUTO_ONLY"
-  source_subnetwork_ip_ranges_to_nat = "LIST_OF_SUBNETWORKS"
-  min_ports_per_vm                   = "8192"
-  tcp_transitory_idle_timeout_sec    = "30"
-  tcp_established_idle_timeout_sec   = "1200"
-  udp_idle_timeout_sec               = "30"
-  icmp_idle_timeout_sec              = "30"
-  subnetwork {
-    name                    = google_compute_subnetwork.subnetwork.self_link
-    source_ip_ranges_to_nat = ["ALL_IP_RANGES"]
-  }
+  region                             = var.region
+
+  nat_ip_allocate_option             = "MANUAL_ONLY"
+  nat_ips                            = google_compute_address.nat_manual_ip.*.self_link
+  source_subnetwork_ip_ranges_to_nat = "ALL_SUBNETWORKS_ALL_IP_RANGES"
+
+  min_ports_per_vm                   = var.min_ports_per_vm
+  tcp_transitory_idle_timeout_sec    = var.tcp_transitory_idle_timeout_sec
+  tcp_established_idle_timeout_sec   = var.tcp_established_idle_timeout_sec
+  udp_idle_timeout_sec               = var.udp_idle_timeout_sec
+  icmp_idle_timeout_sec              = var.icmp_idle_timeout_sec
+
   log_config {
-    filter = "ERRORS_ONLY"
     enable = true
+    filter = "ERRORS_ONLY"
   }
-}
\ No newline at end of file
+}
+
+# TODO: We need the connector if we want GCP Functions and GCP Cloud Run to go through the NAT
```

Comment:
Probably you can remove this part of the "connector" as it was specific to other projects.

Reply:
I Will do, thanks!